immix/llvm/plimmix_pass.cpp

```diff
@@ -49,31 +49,54 @@ namespace
   */
   void immixPassLogic(Module &M)
   {
+    auto isMain = false;
     for (auto FB = M.functions().begin(), FE = M.functions().end(); FB != FE; ++FB)
```

Comment:
it's more common to write this, IIRC. 
```suggestion
    for (auto FB = M.functions().begin(); FB != FE = M.functions().end(); FB++)
```

Reply:
It's copied from LLVM source code... Please do not submit suggestions like this again if it's not severe.